src/mame/barcrest/mpu4.cpp

```diff
@@ -179,7 +179,7 @@ IRQ line connected to CPU
  0F00-0F03 |R/W| D D D D D D D D | PIA6821 IC8
            |   |                 |
            |   |                 | port A
-           |   |                 |
+           |   |       	          |
```

Comment:
There seems to have been a minor accident here.

Reply:
I think my IDE has eaten some spaces, let me see if I can fix it locally and push it back.